build/nuke/Native/SilkDroid.cs

```diff
@@ -36,6 +36,12 @@ string AndroidHome
                 return AndroidHomeValue;
             }
 
+            if ((Environment.GetEnvironmentVariable("ANDROID_HOME") ?? Environment.GetEnvironmentVariable("ANDROID_SDK_ROOT")) is {} sdk)
```

Comment:
for reasons i cannot describe this is the worst thing ive ever seen but i cant think of a better way of writing this line of code in C#